versioned_docs/version-5.x/header-buttons.md

```diff
@@ -35,7 +35,8 @@ function StackScreen() {
 }
 ```
 
-When we define our button this way, the `this` variable in `options` is _not_ the `HomeScreen` instance, so you can't call `setState` or any instance methods on it. This is pretty important because it's extremely common to want the buttons in your header to interact with the screen that the header belongs to. So, we will look how to do this next.
+> Note that there is a community-developed solution for rendering header buttons that takes care of the correct styling - see [react-navigation-header-buttons](https://github.com/vonovak/react-navigation-header-buttons)
```

Comment:
Can you add the library here? https://reactnavigation.org/docs/community-libraries-and-navigators

Reply:
On a sidenote, it'll be cool if the header buttons library used the theme from `react-navigation` so that users could customize the colors in one place, also would be good to use the `tintColor` argument in the examples for the buttons.

Reply:
@satya164 the library is already in the list

> On a sidenote, it'll be cool if the header buttons library used the theme from react-navigation

yeah, that's a good point, I'll add that

Reply:
the lib now takes colors from react navigation's theme :)